packages/block-editor/src/hooks/block-bindings.scss

```diff
@@ -1,3 +1,14 @@
-.components-panel__block-bindings-panel .components-item__block-bindings-source {
-	color: $gray-700;
+div.block-editor-bindings__panel {
+	grid-template-columns: auto;
```

Comment:
For this use case, it seems we need to use just 1 column in the [`ToolsPanel` component](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-toolspanel--docs), but it seems it always uses 2 columns. @WordPress/gutenberg-components did you consider adding a new property to define the number of grid columns? This is something the Grid component is doing, for example.

Apart from that, I had to add more specificity to override those styles.

Reply:
We were not as involved in the `ToolsPanel` component as on other components, and therefore we may lack some context on how the component layout works — but, looking at [the component's docs](https://github.com/WordPress/gutenberg/blob/391aef63dc6020610d58065dc42d94761e280e06/packages/components/src/tools-panel/tools-panel/README.md#toolspanel-layout), it seems like those decisions were taken deliberately.

My guess is that `ToolsPanel` was thought for a very specific case, and at the time it was not considered to allow more flexibility (perhaps on purpose, to ensure more coherency across the block editor's UI).

My advice is to follow the recommendations of the docs, and at most open a separate issue where we can discuss if and how to make `ToolsPanel`'s layout more flexible (by allowing a custom number of columns? by allowing each item to specify a col span ?)

Reply:
So, instead of changing the wrapper to have just one column, we need to change the style of each item to fit both columns?

For reference, other examples where the "one-column" layout is used:

**Colors**
<img width="278" alt="Screenshot 2024-07-30 at 08 45 45" src="https://github.com/user-attachments/assets/91c70be2-4ae1-49f2-a20a-5a658d7fcdb2">

**Dimensions**
<img width="277" alt="Screenshot 2024-07-30 at 08 46 17" src="https://github.com/user-attachments/assets/f7ee681c-3b3d-4069-8c62-d0cf6c56a9c8">

**Filters**
<img width="280" alt="Screenshot 2024-07-30 at 08 46 02" src="https://github.com/user-attachments/assets/8291d94b-c2c1-4994-b7c2-5a3a3139e7b6">

**Border & Shadow**
<img width="282" alt="Screenshot 2024-07-30 at 08 46 34" src="https://github.com/user-attachments/assets/43b9f96e-e40f-4ac2-94a3-85dce24b20d5">